### PR TITLE
chore: v1.0.0-beta.129

### PR DESCRIPTION
**Pull request by tatomyr**
## What/Why/How?

Release v1.0.0-beta.129.

This includes the following:
-  #1123
- #1130
- #1132
- #1133
- #1136
- #1137

## Reference

The related docs changes to be merged: 
- https://github.com/Redocly/redocly-cli/pull/1139

## Check yourself

- [ ] Code is linted
- [x] Tested with redoc/reference-docs/workflows
- [ ] All new/updated code is covered with tests

## Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -5,6 +5,23 @@ toc:
...
+### Fixes
+
+- Fixed build-docs command not working in Docker.
+- Other stability fixes and improvements.
+
+### Changes
+
+- Changed the `push` command interface.
+- Improved Redocly configuration validation.
+- Documentation adn messaging corrections.
```
Comment (RomanHotsiy) [inline docs/changelog.md:23]: Typo

File: docs/changelog.md [hunk 2]
```diff
@@ -5,6 +5,23 @@ toc:
...
+
+- Added telemetry.
+### Fixes
+
+- Fixed build-docs command not working in Docker.
+- Other stability fixes and improvements.
+
+### Changes
+
+- Changed the `push` command interface.
```
Comment (RomanHotsiy) [inline docs/changelog.md:21]: Maybe clarify it's not a breaking change

File: docs/changelog.md [hunk 3]
```diff
@@ -5,6 +5,23 @@ toc:
 
 # Redocly CLI changelog
 
+
+## 1.0.0-beta.129 (2023-06-26)
+
+### Features
+
+- Added telemetry.
```
Comment (lornajane) [inline docs/changelog.md:13]: Can we call this "Added product metrics collection" to be more specific about the purpose?

Comment (tatomyr) [inline docs/changelog.md:13]:
```suggestion
- Added product metrics collection.
```

Comment (tatomyr) [reply docs/changelog.md:13]: Surely!

**Review by IgorKarpiuk — APPROVED** [review]

**Review by lornajane — APPROVED** [review]
Looking great! Thank you :)

**Pull request merged**